sklearn/datasets/samples_generator.py

```diff
@@ -250,8 +250,7 @@ def make_classification(n_samples=100, n_features=20, n_informative=2,
 
 def make_multilabel_classification(n_samples=100, n_features=20, n_classes=5,
                                    n_labels=2, length=50, allow_unlabeled=True,
-                                   sparse=False, return_indicator=False,
```

Comment:
Please don't remove `return_indicator`: we only deprecated its default value, not its existence.